content/en/docs/Reference/build-from-scratch.md

```diff
@@ -37,7 +37,16 @@ This document outlines the steps for making any base image fully bootable with t
 
 ## Prerequisites
 
-To follow the steps below, you'll need to have Docker or a container engine installed on your local machine. Additionally, note that the steps have been tested on Linux but should also work in other environments. If you encounter any issues, please feel free to open up issues and help us improve the Documentation!
+To follow the steps below, you'll need to have Docker or a container engine installed on your local machine. Additionally, note that the steps have been tested on Linux but should also work in other environments. 
+
+The base OS used as an example in this document is Fedora 38. However, the steps should work with any other OS as well. The OS should meet the following requirements:
+
+- Use `systemd` (`openRC` is currently not supported, but in the roadmap)
```

Comment:
maybe worth a link to the roadmap? not sure if we have a card for the pure alpine

Reply:
link added :+1: